firestore-bigquery-export/functions/package.json

```diff
@@ -17,7 +17,7 @@
     "@google-cloud/bigquery": "^2.1.0",
     "@types/chai": "^4.1.6",
     "chai": "^4.2.0",
-    "firebase-admin": "^9.1.0",
+    "firebase-admin": "^8.1.0",
```

Comment:
@russellwheatley is this to undo the type issues we talked about in the meeting today?

Reply:
Yes, it was causing problems in the BQ tests.

Reply:
mind moving this and the firestore-bigquery-change-tracker change into another PR to make it a little easier to track?

Reply:
Yeah, that makes sense. Will do 😄